doc/ref_cert/lfn/chapters/chapter01.md

```diff
@@ -4,14 +4,39 @@
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
-* [1.1 Overview](#1.1)
+* [1.1 Introduction](#1.1)
 * [1.2 Scope](#1.2)
-* [1.3 Resources & References](#1.3)
+* [1.3 Principles and Guidelines](#1.3)
+  * [1.3.1 Overarching Objectives and Goals](#1.3.1)
+* [1.4	Best Practices](#1.4)
+* [1.5 Verification methodologies](#1.5)
+*	[1.6 Assumptions & Dependencies](#1.6)
+*	[1.7 Results Collation & Presentation](#1.7)
+*	[1.8 Governance](#1.8)
+* [x.x Resources & References](#x.x)
+
+## Synopsis
+
+Ensure Reference Implementation of CNTT Reference Model and CNTT Reference Architecture meets industry driven quality assurance standards for compliance, verification and validation.  The OPNFV Verified Program (OVP), by Linux Foundation Networking (LFN), overseen by the Compliance Verification Committee (CVC), will provide tracking and governance for RM/RA verification.
+
+For the purposes of this chapter, NFVI+VNF testing will be performed for **Verification**, **Validations**, ultimately leading to **Certifications** indicating a measured performance of the adherence to, and demonstrated proficiency with, all aspects of software delivery including but no limited to planning, logistics for communication, and testing of installed, or implmented, NFVI.
 
```

Comment:
thanks for describing the intention of cerification. And is it ok to explicitly call out the VNF part, as something like below:  Similar to NFVI certification, its important to make sure that VNF is compliance to the refernce model defined in RM chapters and certify the VNF compliance, verification and validation on top of CNTT compliance NFVI.

Also compliance part is missing at line 22, its good to add them as well into the list.
thanks

Reply:
Thank you @mkr1481 for the Feedback.  I've added the following to the 2nd paragraph of the synopsis.

Please let me know if we can close (Resolve) this conversation.

_Similarily, VNFs will be checked for **Compliance** and **Validations** against the Reference Model (RM) and Reference Architecture (RA) requirements ensuring VNF instantiation, stability, and successful removal of the VNF from the implementation of the stated architecture.  The additional VNF testing will verify that the developed VNF meets prescribed RM/RA infrastructure standards._

Reply:
thanks mike